SU-SWS/react_paragraphs
# Fix media browser form

# READY FOR REVIEW

# Summary
- When the media browser is open, the text entry fields are not interactive. This break when trying to add alt text or document name after uploading a new file.
- We should start to look at setting up some functional tests for this reason.

# Need Review By (Date)
- asap

# Urgency
- pretty high.

# Steps to Test
1. checkout this branch
1. clear browser cache
1. create a page and either in a wysiwyg or a media field upload a new image/document
1. after uploading, verify you can change the alt text or the title of the document.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
